frontend/public/components/factory/table.tsx

```diff
@@ -145,6 +146,7 @@ const sorts = {
   machinePhase: (machine: MachineKind): string => getMachinePhase(machine),
   nodePods: (node: NodeKind): number => nodePods(node),
   pvcUsed: (pvc: K8sResourceKind): number => pvcUsed(pvc),
+  volumeSnapshotSize: (snapshot: K8sResourceKind): number => snapshotSize(snapshot),
```

Comment:
seems like this should be 
```suggestion
  volumeSnapshotSize: (snapshot: VolumeSnapshotKind): number => snapshotSize(snapshot),
```